specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -3756,6 +3844,173 @@
               "Desktop"
             ]
           }
+        }
+      }
+    },
+    "SocketAddressMatchConditionParameters": {
+      "description": "Defines the parameters for SocketAddress match conditions",
+      "required": [
+        "operator",
+        "typeName"
+      ],
+      "properties": {
+        "typeName": {
+          "type": "string",
+          "enum": [
+            "DeliveryRuleSocketAddressConditionParameters"
+          ]
+        },
+        "operator": {
+          "description": "Describes operator to be matched",
+          "type": "string",
+          "enum": [
+            "Any",
+            "IPMatch",
+            "GeoMatch"
```

Comment:
remove geomatch. See MatchConditionValidator.cs in shared-protection for details.

Reply:
removed